src/pages/Homepage.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import HeroComponent from '../components/HeroComponent';
+
+import { getCrypto } from '../redux/currencies/currenciesSlIce';
+import SingleCurrency from '../components/SIngleCurrency';
+
+const Homepage = () => {
+  const { curencies } = useSelector((state) => state.currencies);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCrypto());
+  }, [dispatch]);
+
+  return (
+    <section className="section-container">
+      <HeroComponent info="common cryptocurrencies" />
+      <div className="home-intros">
+        <div className="crypto-img-container " />
+        <article className="home-info">
+          <h2> Crypto-Currencies </h2>
+        </article>
+      </div>
+      <h3>Stats per rank</h3>
+      <div className="home-container">
+        {curencies.map(({
+          name, rank, id, symbol,
+        }) => (
+          <SingleCurrency
```

Comment:
- [ ] The list of coins in the `Home page` can be filtered using given parameters (like the name of the coin for example), kindly make sure to add a search area on the home page that can filter the list of coins and display only relevant ones when searching